crates/egui/src/ui.rs

```diff
@@ -2142,7 +2212,10 @@ impl Ui {
         let mut child_ui = self.new_child(ui_builder);
         self.next_auto_id_salt = next_auto_id_salt; // HACK: we want `scope` to only increment this once, so that `ui.scope` is equivalent to `ui.allocate_space`.
         let ret = add_contents(&mut child_ui);
-        let response = self.allocate_rect(child_ui.min_rect(), Sense::hover());
+        #[allow(deprecated)]
+        let response = child_ui.interact_bg();
+        child_ui.should_interact_bg_on_drop = false;
+        self.allocate_rect(child_ui.min_rect(), Sense::hover());
```

Comment:
I also wonder if we should replace this with `Ui::advance_cursor_after_rect` since we don't need this response?

Reply:
Yeah, makes sense

Reply:
Fixed in https://github.com/emilk/egui/pull/5130